src/vconsole/vconsole-setup.c

```diff
@@ -82,7 +84,8 @@ static bool is_settable(int fd) {
          *
          * http://lists.freedesktop.org/archives/systemd-devel/2013-February/008573.html
          */
-        return r == 0 && IN_SET(curr_mode, K_XLATE, K_UNICODE);
+        return r < 0 ? -errno :
+               IN_SET(curr_mode, K_XLATE, K_UNICODE) ? 0 : (errno = EBUSY, -EBUSY);
```

Comment:
No, please don't do that. That's not easy to read — just use normal `if`s.

Don't set errno, we never do that.